streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java

```diff
@@ -1158,6 +1159,75 @@ public void shouldSkipOnDeserializationErrorsWhenReprocessing() {
         assertEquals(0, stateRestoreCallback.restored.size());
     }
 
+    @SuppressWarnings("deprecation")
+    @Test
+    public void verifyExceptionHandlerAcceptNewConfigWhenBothArePresent() {
```

Comment:
This test raised the question if we should log a WARN if both are set?

But why do we test this on `GlobalStateManagerImplTest`? Seems this should go into `StreamsConfigTest`?

Reply:
Moved test to StreamsConfigTest, agree.